tmt/steps/report/junit.py

```diff
@@ -74,13 +74,13 @@ def make_junit_xml(report: "tmt.steps.report.ReportPlugin") -> JunitTestSuite:
             stdout=main_log)
         # Map tmt OUTCOME to JUnit states
         if result.result == tmt.base.ResultOutcome.ERROR:
-            case.add_error_info(result.result.value)
+            case.add_error_info(result.result.value, output=result.failures(main_log, 'ERROR'))
         elif result.result == tmt.base.ResultOutcome.FAIL:
-            case.add_failure_info(result.result.value)
+            case.add_failure_info(result.result.value, output=result.failures(main_log))
         elif result.result == tmt.base.ResultOutcome.INFO:
-            case.add_skipped_info(result.result.value)
+            case.add_skipped_info(result.result.value, output=result.failures(main_log, 'INFO'))
         elif result.result == tmt.base.ResultOutcome.WARN:
-            case.add_error_info(result.result.value)
+            case.add_error_info(result.result.value, output=result.failures(main_log, 'WARN'))
```

Comment:
Actually, in `beakerlib` individual asserts would still be marked as `FAIL` even if the cleanup phase end result is `WARN`. I'd say, for `beakerlib` tests we should always look for `FAIL` asserts. @kkaarreell, what do you think about this?

Reply:
Yes, always